homeassistant/components/ohme/entity.py

```diff
@@ -0,0 +1,38 @@
+"""Base class for entities."""
+
+from homeassistant.core import callback
+from homeassistant.helpers.entity import Entity
+
+
+class OhmeEntity(Entity):
```

Comment:
```suggestion
class OhmeEntity(Entity):
```
those usally inherit from `CoordinatorEntity`